airflow/www/templates/airflow/dags.html

```diff
@@ -97,8 +97,8 @@ <h2>DAGs</h2>
                 </td>
 
                 <!-- Column 6: Recent Tasks -->
-                <td style="padding:0px; width:200px; height:10px;">
-                    <svg height="10" width="10" id='task-run-{{ dag.safe_dag_id }}' style="display: block;"></svg>
+                <td class="td-recent-tasks">
```

Comment:
```suggestion
                <td class="recent-tasks">
```

and then `td.recent-tasks` as the classname - no need to put information that is already available in a native CSS selector in the class name.

Reply:
I agree. I will update a PR..